src/vivarium_cluster_tools/psimulate/cli.py

```diff
@@ -82,12 +82,29 @@ def psimulate():
     "configuration file.",
     callback=cli_tools.coerce_to_full_path,
 )
+# TODO: MIC-5234 - use this where other cli cluster options are being used
+@click.option(
```

Comment:
We'll want this in the other commands as well (expand, restart, etc). Should probably add it to `shared_options` via `cli_tools`

Reply:
It gets attached to the sim so it should be in those sims since it gets put in the simulation instance.